subjects/union.en.md

```diff
@@ -4,17 +4,17 @@
 
 Write a program that takes two `string` and displays, without doubles, the characters that appear in either one of the `string`.
 
-The display will be in the order characters appear in the command line, and will be followed by a newline (`'\n'`).
+The display will be in the order that the characters will appear on the command line and will be followed by a newline (`'\n'`).
 
 If the number of arguments is different from 2, then the program displays newline (`'\n'`).
 
 ### Usage
 
 ```console
 student@ubuntu:~/[[ROOT]]/union$ go build
-student@ubuntu:~/[[ROOT]]/union$ ./union zpadinton "paqefwtdjetyiytjneytjoeyjnejeyj" | cat -e
+student@ubuntu:~/[[ROOT]]/union$ ./union "zpadinton" "paqefwtdjetyiytjneytjoeyjnejeyj" | cat -e
```

Comment:
Why the quotes ?

Reply:
I added the quotes to identify the two strings, otherwise it can be a little bit confusing for the students. 
Do you think it is better without quotes?

Reply:
These are not strings, but arguments, these quotes do not have the same meaning as in Go.

Reply:
ok. i will take them all of. thank you